pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -1387,6 +1387,61 @@ class ClassTypeHintMatch:
             ],
         ),
     ],
+    "humidifier": [
+        ClassTypeHintMatch(
+            base_class="Entity",
+            matches=_ENTITY_MATCH,
+        ),
+        ClassTypeHintMatch(
+            base_class="ToggleEntity",
+            matches=_TOGGLE_ENTITY_MATCH,
+        ),
+        ClassTypeHintMatch(
+            base_class="HumidifierEntity",
```

Comment:
I only wanted to add `supported_features`, but I found that the whole HumidifierEntity was missing from the pylint plugin.